src/app/clusters/webrtc-transport-requestor-server/webrtc-transport-requestor-server.h

```diff
@@ -87,7 +90,7 @@ class WebRTCTransportRequestorDelegate
      * @return CHIP_ERROR
      *   - Returns error if the session is invalid or the candidates cannot be processed
      */
-    virtual CHIP_ERROR HandleICECandidates(uint16_t sessionId, const std::vector<std::string> & candidates) = 0;
+    virtual CHIP_ERROR HandleICECandidates(uint16_t sessionId, const std::vector<ICECandidateStruct> & candidates) = 0;
```

Comment:
Again, please fix the documentation.  I have not checked whether the callees handle this correctly, by the way... This is where the PR is hard to review because it's hard to find those pieces.  Certainly WebRTCProviderClient::ProvideICECandidates looks like use-after-free... Can't speak for other bits that handle ICECandidateStruct.